starter-dev-backend/src/schema/github/github.typedefs.ts

```diff
@@ -21,14 +21,14 @@ export const githubTypeDefs = gql`
   """
   A Owner object used in Repo
   """
-  type Owner {
+  type User {
     login: String
   }
 
   """
   A User object
   """
```

Comment:
Suggestion: Update comment to reflect schema change.

```suggestion
  """
  An Owner object
  """
```